setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.25.3
+current_version = 0.26.0
```

Comment:
I think it's redundant here - version will be bumped in the PR into main created by the github actions bot.